pr_agent/tools/pr_line_questions.py

```diff
@@ -56,6 +60,10 @@ async def run(self):
         # if get_settings().config.publish_output:
         #     self.git_provider.publish_comment("Preparing answer...", is_temporary=True)
 
+        # set conversation history if enabled
+        if self.use_conversation_history:
+            self._load_conversation_history()
```

Comment:
Please add another condition for GitHub provider or, if you're interested, you can expand the code further to support other providers. We also need a short comment here explaining that only GitHub is supported at this stage.
```suggestion
        # set conversation history if enabled
        # currently only supports GitHub provider
        if self.use_conversation_history and isinstance(self.git_provider, GithubProvider):
            self._load_conversation_history()
```